### ISSUE TITLE
Accessibility: Failed Dropdown Accessibility Tests

### ISSUE THREAD

**Issue by clepore** [infor-design/enterprise#404]
Axe is reporting the following WCAG2AA errors as critical errors

- [x] Ensures ARIA attributes are allowed for an element\'s role (remove aria-readonly)
- [x] Ensures elements with an ARIA role that require child roles contain them (doesn't like the div and would prefer using an input)
- [x] Ensures all ARIA attributes have valid values (remove aria-controls and add only when the list is actually open.
- [ ] Test and document use with Jaws/nvda

Follow debugging instructions on docs/TESTING.md to investigate the issue in the devTools console, and run test in high-contrast which has disables zero rules, These errors can also be double checked with Chrome Audit in devTools, or use the Chrome or Firefox extension, https://www.axe-core.org/

This also effects the dependent on dropdown; multiselect component as well

Formerly: SOHO-7937

**Comment by tmcconechy** [comment]
Approach here will be to try to change from a combobox pattern to https://w3c.github.io/aria-practices/examples/listbox/listbox-collapsible.html

**Comment by janahintal** [comment]
QA Passed. Will Moved to Done. 
Tested in: http://master-enterprise.demo.design.infor.com/components/dropdown/example-index.html using Chrome and ChromeVox.
